### PR TITLE
rationale

### LINKED ISSUE #2: rationale
Hey @leobalter! Great stuff :D 

I think it would be cool to add to the README a tiny "rationale" section, letting people know **why** this project exists and how does it compare to other testing frameworks out there.

Abraço! \o/

### PR DESCRIPTION
**Pull request by leobalter**
Close #2 

cc @garu, review, please?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+
+- Classes
+- Promises
+- Generators (++ to this one as it solved the async tests order as a charm)
+- Arrow functions
+- template strings
+- parameters destructuring
+- shorthand properties
+
+Reviews, comments and PR are more than welcoming! I'll love any feedback.
```
Comment (garu) [inline README.md]: minor typo: should be "Reviews, comments and PRs are more than welcome!"

File: README.md [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+
+## Rationale
+
+I saw a lot of people very excited with the new changes on [babel (6to5)](https://babeljs.io/) and after talking to some friends at work, I decided to play with some ES6 doing this unit test framework using a lot of its features.
+
+So far it's using several ES6 features together, like:
+
+- Classes
+- Promises
+- Generators (++ to this one as it solved the async tests order as a charm)
```
Comment (garu) [inline README.md]: minor typo: should be "like a charm"

**Comment by garu** [comment]
Tá lindão! LGTM! \o/

**Comment by leobalter** [comment]
@garu++

**Pull request merged**